makefile

```diff
@@ -126,55 +133,34 @@ build_js:
 mockAgent:
 	cd acceptance/mockAgent && $(GO) build $(GOBUILD_FLAGS) ${LDFLAGS}
 
-GOVET     = $(GOBIN)/govet
-GOTOOLS_SRC = golang.org/x/tools
-
-GOVET_EXCLUDE_DIRS = Godeps/ build/ chef/ vagrant/ vendor/
```

Comment:
At this point, can we delete the Godeps, chef and vagrant directories from the develop branch?